### PR TITLE
Import script for St Helens (2024-05-02) (closes #6517)

### LINKED ISSUE #6517: Import SHN-St Helens Metropolitan Borough Council for 2024-05-02
EMS: Xpress DC
Files:
- `SHN/2024-05-02/2024-03-08T16:27:14.461070/Democracy_Club__02May2024 - updated.tsv`

### PR DESCRIPTION
**Pull request by polling-bot-4000**
PR triggered by upload at https://github.com/DemocracyClub/UK-Polling-Stations/issues/6517

### CODE REVIEW

**Comment by awdem** [comment]
https://app.asana.com/0/1205951895246773/1206793583411963/f

**Comment by awdem** [comment]
Tagged this for review because it's updating live data, but I'm not sure if we should wait for a response from the council RE the card above before merging or not?

**Review by GeoWill — APPROVED** [review]

**Pull request merged**
